src/bench/stream.zig

```diff
@@ -100,6 +109,9 @@ pub fn main() !void {
         .@"gen-ascii" => try genAscii(writer, seed),
         .@"gen-utf8" => try genUtf8(writer, seed),
         .@"gen-rand" => try genRand(writer, seed),
+        .@"gen-osc" => try genOsc(writer, seed, 0.5),
```

Comment:
Note `0.5` is a pretty unrealistic workload, but it gives us a ballpark of "average" performance if this were fed into an OSC parser. I think realistically OSC sequences are at least 90% valid (but likely closer to 99%). If we were benchmarking, I think it'd be cleaner to separate out 100% valid and 100% invalid sources rather than use this mixed version.